config/model_configs/gpu_amip_albedo_static_map.yml

```diff
@@ -0,0 +1,22 @@
+alpha_rayleigh_uh: 0
+alpha_rayleigh_w: 10
+apply_limiter: false
+dt: "150secs"
+dt_cpl: 150
+dt_rad: "1hours"
+dt_save_to_sol: "1days"
+dz_bottom: 30
+dz_top: 3000
+h_elem: 16
```

Comment:
Could we mimic the [functional albedo resolution config](https://github.com/CliMA/ClimaCoupler.jl/blob/js/gpu-amip-albedo/config/model_configs/gpu_amip_albedo_function.yml), or do these break for the lower resolution? 

Also, I noticed that the [temporal albedo slabplanet](https://github.com/CliMA/ClimaCoupler.jl/blob/js/gpu-amip-albedo/config/model_configs/gpu_slabplanet_albedo_temporal_map.yml) has a higher resolution than the other two albedo runs. Is it because it was breaking for helem=4?

Reply:
Good catch! I'll lower them to 4, I think it should be fine.

The temporal albedo slabplanet has a higher h_elem because I copied these from the non-GPU slabplanet runs, where the functional and static map albedo have `h_elem: 4` and temporal has `h_elem: 6`. I'm not sure why this is, and happy to change them all to 4.

Reply:
Hmm, maybe it is unstable. 🤔 Worth trying though.

Reply:
they all work with the lower resolution :)